modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectActionLocalServiceImpl.java

```diff
@@ -343,7 +343,8 @@ private void _validatePredefinedValues(
 
 			if (!objectField.isRequired() ||
 				Validator.isNotNull(
-					predefinedValuesMap.get(objectField.getName()))) {
+					predefinedValuesMap.get(objectField.getName())) ||
+				Validator.isNotNull(objectField.getRelationshipType())) {
```

Comment:
@daiazius your fix likes good to me. I believe we can be a little more clear here if we check the BusinessType rather than the RelationshipType.

It would be something like this:
`StringUtil.equals(businessType, ObjectFieldConstants.BUSINESS_TYPE_RELATIONSHIP)`

What do you think about this approach?

Thank you in advance